abz/acousticbrainz.py

```diff
@@ -93,7 +93,18 @@ def submit_features(recordingid, features):
     featstr = json.dumps(features)
 
     host = config.settings["host"]
-    url = compat.urlunparse(('http', host, '/%s/low-level' % recordingid, '', '', ''))
+    if sys.version_info < (3, 0):
+        try:
+            import urllib3.contrib.pyopenssl
+            urllib3.contrib.pyopenssl.inject_into_urllib3()
+        except ImportError:
+            pass
+    if host[:9] == "localhost": 
```

Comment:
instead of specifically checking against localhost, I think that we should allow people to specify a protocol in the config file. I would do the following things:
 - Change the default value in the config file to `https://acousticbrainz.org`
 - In this method, if there is a protocol set (`http://` or `https://`) then use that. If there is no protocol set (e.g. the user may have overwritten this to be `acousticbrainz.org`), default to https
 - We don't need to use `urlunparse` to construct this url. If the protocol is already there we might just want to use string concatenation/formatting to go "%s/%s/low-level" % (host, recordingid) or similar.

Reply:
while we're working on this method we should change the endpoint to the endpoint in the `/api` url structure: https://acousticbrainz.readthedocs.io/api.html#post--api-v1-(uuid-mbid)-low-level